subprojects/logging/src/main/java/org/gradle/internal/logging/sink/ProgressLogEventGenerator.java

```diff
@@ -16,9 +16,13 @@
 
 package org.gradle.internal.logging.sink;
```

Comment:
Do we think that merging both renderers might increase complexity of that one class? Is there potentially a way to pull out some of the logic? Maybe the logic that holds and manages the buffer state?